SU-SWS/stanford_paragraph_types
# EARTH-401 Simple Slat Paragraph type

# READY FOR REVIEW

# Summary
- Simple slat (Editorial Sidebar) paragraph type.

# Needed By (Date)
- End of sprint

# Urgency
- Medium

# Steps to Test

1. Checkout branch and same branch in [stanford_components](https://github.com/SU-SWS/stanford_components/pull/79)
2. clear cache. enable stanford_paragraph_slat module
4. create complex page with "Editorial Sidebar" component
5. verify it looks good enough for now.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/79

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
